### ISSUE TITLE
mime check for urls revisited

### ISSUE THREAD

**Issue by majkinetor** [majkinetor/au#83]
Perhaps MIME check should work only on URL32/64 urls, and not on others.

That would allow for extra URLs to be checked for existence: ReleaseNotes, Manuals etc.

Update option `NoMimeCheck` doesn't sound like an idea because you certainly want it for subset of URLs.

/cc @AdmiringWorm

**Comment by AdmiringWorm** [comment]
Personally I think it's fine if AU checks the Mime Type for every url variable that starts with `URL`.
But, just thinking out loud, perhaps adding something like `URLNAMEDisableCheck = $true` and try detecting that could alleviate some pain.

**Comment by majkinetor** [comment]
Well, I see now that `iwr -Method Head` does the same thing as integrated AU stuff, so its actually not a big deal to. 

`URLNAMEDisableCheck = $true` I don't like, because its extra thing, but some special char could do the trick such as `URL_`.

So 

```
$Latest = {
   URL32 = ... # check mime and existence 
   URL_32 = ... # do not check mime but check existence 
   MyUrl = ... # do not check anything
}
```

**Comment by AdmiringWorm** [comment]
I agree, using a special character to disable the mime check would be better.

**Comment by FranklinYu** [comment]
If this is implemented, we can also check links in `*.nuspec` such as `projectUrl` and `licenseUrl`.
